config/locales/diaspora/en.yml

```diff
@@ -1365,3 +1366,20 @@ en:
       default: "The secret code did not match with the image"
       user: "The secret image and code were different"
       failed: "Human verification failed"
+
+  statistics:
+    name: "Name"
+    network: "Network"
+    services: "Available services"
```

Comment:
`None` for the disabled state just sounds silly. I still think that this is not explanatory enough for the ordinary user, especially since the page is also targeted at users in search for their first home pod.

Additionally it's currently two pro hiding disabled services and two against it, we need another opinion here.